appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -517,7 +519,8 @@ class LoggedInFlowNode(
                 serverNames = serverNames,
                 trigger = trigger,
                 initialElement = RoomNavigationTarget.Messages(
-                    focusedEventId = eventId
+                    focusedEventId = eventId,
+                    inThreadId = threadId,
```

Comment:
I would expect to have a function like `attachThread`?

Reply:
I don't think we need one because we'd just duplicate most of the logic, unless we want to open a thread *without* opening the room instead.